prow/cmd/deck/main.go

```diff
@@ -499,8 +503,9 @@ func prodOnlyMain(cfg config.Getter, o options, mux *http.ServeMux) *http.ServeM
 		mux.Handle("/tide-history.js", gziphandler.GzipHandler(handleTideHistory(ta)))
 	}
 
+	var pcfg *plugins.Configuration
```

Comment:
This is used 100 lines later, consider making this pluginConfig or something easier to understand.

https://dave.cheney.net/practical-go/presentations/qcon-china.html#_identifier_length